Include sys/regset.h for WasmSignalHandlers as it is not found otherwise
--- firefox-68.6.0/js/src/wasm/WasmSignalHandlers.cpp.orig      2020-03-19 22:53:45.577402027 +0000
+++ firefox-68.6.0/js/src/wasm/WasmSignalHandlers.cpp   2020-03-19 22:53:16.349533586 +0000
@@ -22,6 +22,7 @@
 #include "mozilla/ScopeExit.h"
 #include "mozilla/ThreadLocal.h"
 
+#include <sys/regset.h>
 #include "threading/Thread.h"
 #include "vm/Realm.h"
 #include "vm/Runtime.h"
